src/services/userService.js

```diff
@@ -1,30 +1,15 @@
 const mongoose = require('mongoose');
 mongoose.Promise = require('bluebird');
-const User = require('mongoose').model('User');
+const User = mongoose.model('User');
 
-let userService = logger => {
+let userService = () => {
     let getAll = () => {
-        return new Promise((resolve, reject) => {
-            return User.find((err, results) => {
-                if (err) {
-                    logger.error(err);
-                    reject({error: err});
-                } else
-                    resolve(results);
-            });
-        });
+        return User.find().exec();
```

Comment:
You'll read about what `exec()` does later on. But basically, it returns a promise, and we're already plugin in Bluebird as the mongoose promise object (`mongoose.Promise = require('bluebird')`). So there's no need to wrap a promise inside a promise. For more info, [click here](http://mongoosejs.com/docs/promises.html).